src/algorithms/banditpam.cpp

```diff
@@ -319,6 +340,9 @@ namespace km {
       }
       updated_sigma(k, n) = arma::stddev(sample);
     }
+    // reset batchSize the original batch size as it's a global variable
```

Comment:
sorry I should have added "to" between "batchSize" and "the original batch size"